hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestContainerEndpoint.java

```diff
@@ -629,19 +642,20 @@ public void testGetContainersWithPrevKey()
 
     ContainersResponse.ContainersResponseData data =
         responseObject.getContainersResponseData();
-    // Ensure that the total count of containers is 4
-    assertEquals(4, data.getTotalCount());
+    // Ensure that the total count of containers is 3 as containers having
+    // ID's 1,2, will be skipped and the next 3 containers will be returned
+    assertEquals(3, data.getTotalCount());
```

Comment:
can you pls assert prevKey also here ?

Reply:
Thanks for the comment @devmadhuu 
I have added assertions for non-sequential container Ids also